src/nagini_translation/main.py

```diff
@@ -75,7 +77,8 @@ def load_sil_files(jvm: JVM, sif: bool = False):
 
 
 def translate(path: str, jvm: JVM, selected: Set[str] = set(),
-              sif: bool = False, reload_resources: bool = False):
+              sif: bool = False, ignore_global: bool = False,
```

Comment:
Just an advice, no actions required here: when modifying a function signature with default values, better not change the order of parameter (particularly if the new parameter has the same type as its neighbors). In this case here, I would advise to put `ignore_globals` in the end since it is new, preventing existing calls to inadvertently pass values intended to ignore_globals instead of to `reload_resources`.

Reply:
Yeah makes sense. I didn't do it here because the options selected, sif and ignore global kind of belong together (they affect what the encoding does) and the reload_resources option is more of a technical detail. I did check all calls of this function to make sure the problem you mentioned doesn't occur. If you don't mind, I'd prefer to leave it the way it is now.